sdk/python/README.md

```diff
@@ -1,11 +1,74 @@
 # Dagger Python SDK
 
-Install [hatch](https://hatch.pypa.io/latest/install/). Recommend [`pipx`](https://github.com/pypa/pipx)), e.g. `pipx install hatch`
+A client package for running [Dagger](https://dagger.io/) pipelines.
 
-Run tests with `hatch run test`. Assumes `go run ./cmd/cloak dev --workdir sdk/python` is running.
+## What is the Dagger Python SDK?
+
+The Dagger Python SDK contains everything you need to develop CI/CD pipelines in Python, and run them on any OCI-compatible container runtime.
+
+## Example
+
+```python
+# say.py
+import sys
+import anyio
+
+import dagger
+
+
+async def main(args: list[str]):
+    async with dagger.Connection() as client:
+        # build container with cowsay entrypoint
+        # note: this is reusable, no request is made to the server
+        ctr = (
+            client.container()
+            .from_("python:alpine")
+            .exec(["pip", "install", "cowsay"])
+            .with_entrypoint(["cowsay"])
+        )
+
+        # run cowsay with requested message
+        # note: methods that return a coroutine with a Result need to
+        # await query execution
+        result = await ctr.exec(args).stdout().contents()
+
+        print(result)
+
+
+if __name__ == "__main__":
+    anyio.run(main, sys.argv[1:])
+```
+
+Run with:
+
+```console
+$ python say.py "Simple is better than complex"
+  _____________________________
+| Simple is better than complex |
+  =============================
+                             \
+                              \
+                                ^__^
+                                (oo)\_______
+                                (__)\       )\/\
+                                    ||----w |
+                                    ||     ||
+```
```

Comment:
I followed this example and it failed as follows:

<img width="818" alt="image" src="https://user-images.githubusercontent.com/3342/200362855-e768f512-69bd-46f0-aa9e-802c5cc6fcf1.png">

Could we add what I missed to this README? It seems incomplete without it.